src/pages/authentication/forgotPass/forgotPass.jsx

```diff
@@ -0,0 +1,32 @@
+import { Navbar } from "../../../components/navbar/navbar"
+import '../forgotPass/forgotPass.css'
```

Comment:
export these components from single index file for easy access

Reply:
ok, i will do that